local-cli/runIOS/runIOS.js

```diff
@@ -82,9 +82,11 @@ function _runIOS(argv, config, resolve, reject) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
+  const xcodeBuildProcess = child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'pipe'});
```

Comment:
Yes, that's the tradeoff :-) I will check if we can get it elsewhere, but it depends on the chosen scheme and therefore we have to read it from the xcode project (which is possible, somehow).

The other option would be to use spawn (async) and wrap it with something that makes it async. Then we could show the output directly and save it to a variable at the same time.